buildpipeline/alpine.3.6.groovy

```diff
@@ -0,0 +1,17 @@
+@Library('dotnet-ci') _
+
+// Incoming parameters.  Access with "params.<param name>".
+// Note that the parameters will be set as env variables so we cannot use names that conflict
+// with the engineering system parameter names.
+// CGroup - Build configuration.
+// TestOuter - If true, runs outerloop, if false runs just innerloop
+
+simpleDockerNode('microsoft/dotnet-buildtools-prereqs:alpine-3.6-3148f11-20171119021156') {
+    stage ('Checkout source') {
+        checkoutRepo()
+    }
+
+    stage ('Build Product') {
+        sh "./build.sh -ConfigurationGroup=${params.CGroup} -TargetArchitecture=x64 -PortableBuild=false -strip-symbols -SkipTests=false -- /p:OutputRid=alpine.3.6-x64 /p:DotNetRestoreSources="
```

Comment:
What's the purpose of setting the restore sources to empty here?

Reply:
It is not needed and I've removed it. I have used a command line that we use in our official builds where we do have this argument, pass in a variable, but that variable is empty. Maybe we don't need it in the official builds either.